src/services/swork.ts

```diff
@@ -31,21 +31,29 @@ export async function reportWorks(
   req: Request
 ) {
   logger.info(`⚙️ [swork]: Call report works with ${JSON.stringify(req.body)}`);
-  const bn = Number(req.body['block_height']);
+  const slot = Number(req.body['block_height']);
   const pk = '0x' + req.body['pub_key'];
   const tx = api.tx.swork.reportWorks(
     pk,
     '0x' + req.body['pre_pub_key'],
-    bn,
+    slot,
     '0x' + req.body['block_hash'],
     req.body['reserved'],
     req.body['files_size'],
     req.body['added_files'].map((file: any) => {
-      const rst: [string, number] = ['0x' + file.hash, file.size];
+      const rst: [string, number, number] = [
+        '0x' + file.hash,
+        file.size,
+        file.c_block_num,
+      ];
       return rst;
     }),
     req.body['deleted_files'].map((file: any) => {
-      const rst: [string, number] = ['0x' + file.hash, file.size];
+      const rst: [string, number, number] = [
```

Comment:
duplicated code with lines above

Reply:
yes, but 1 is `added`, another is `deleted`